src/lib/media-import/status.js

```diff
@@ -222,6 +240,20 @@ ${ maybeExitPrompt }
 		progressTracker.stopPrinting();
 
 		setProgressTrackerSuffix();
+		progressTracker.print();
+
+		const fileErrors = results.failureDetails?.fileErrors;
+		if ( !! fileErrors && fileErrors.length > 0 ) {
+			progressTracker.suffix += `${ chalk.yellow( `⚠️ ${ fileErrors.length } file error(s) found` ) }`;
+			const formattedData = buildFileErrors( fileErrors, exportFileErrorsToJson );
+			const errorsFile = `media-import-${ app.name }-${ Date.now() }${ !! exportFileErrorsToJson ? '.json' : '.txt' }`;
+			try {
+				await fs.writeFile( errorsFile, formattedData );
+				progressTracker.suffix += `\n${ chalk.yellow( `All errors have been exported to ${ chalk.bold( path.resolve( errorsFile ) ) }` ) }`;
+			} catch ( writeFileErr ) {
+				progressTracker.suffix += `\n${ chalk.red( `Could not export errors to file\n${ writeFileErr }` ) }`;
+			}
+		}
```

Comment:
Do you think we could use a line break at the end, to tackle this:

![Screenshot from 2021-04-08 13-59-35](https://user-images.githubusercontent.com/6948790/113990150-fcce3c00-9872-11eb-9066-96dc07a4154e.png)

Reply:
Fixed in 8a18a59, thanks!